package.nix

```diff
@@ -1,23 +1,23 @@
 { stdenv
 , lib
-, naersk
 , pkg-config
 , openssl
 , zlib
+, rustPlatform
 
-# darwin dependencies
+  # darwin dependencies
 , darwin
 }:
 
-naersk.buildPackage {
+rustPlatform.buildRustPackage {
```

Comment:
What's the reason for getting rid of naersk? I'm not a big fan of `buildRustPackage` since it's very annoying to keep the hash updated and takes a long time to calculate it.

Reply:
Yeah I did this reluctantly because naersk doesn't support downloading git dependencies in pure mode. The strange thing is that this didn't seem to affect hydra so I didn't notice it right away. But if anyone needed to build the nix package locally it would fail. I did [request](https://github.com/serokell/deploy-rs/pull/104#issuecomment-892803736) that they publish deploy-rs on crates.io since they merged my changes so we'll see.

I also wanted to give crates2nix a go and see if perhaps that would work.

Reply:
Ah yeah, the long-standing issue of recursive git dependencies has been a constant pain in our other rust repos too... the easy way is to simply add the dependency to our own and naersk will resolve it just fine.

Reply:
looks like nix-community/naersk#167 actually resolves the issue. I'll force push an update that removes my old fix and bumps naersk to this PR instead.